modules/simple_forms_api/app/services/simple_forms_api/pdf_stamper.rb

```diff
@@ -225,16 +224,32 @@ def self.stamp_submission_date(stamped_template_path, config)
         page_configuration = default_page_configuration
         page_configuration[config[:page_number]] = { type: :text, position: date_title_stamp_position }
 
-        multistamp(stamped_template_path, SUBMISSION_DATE_TITLE, page_configuration, 12)
+        verified_multistamp(stamped_template_path, SUBMISSION_DATE_TITLE, page_configuration, 12)
 
         page_configuration = default_page_configuration
         page_configuration[config[:page_number]] = { type: :text, position: date_text_stamp_position }
 
-        multistamp(stamped_template_path, Time.current.in_time_zone('UTC').strftime('%H:%M %Z %D'), page_configuration,
-                   12)
+        current_time = Time.current.in_time_zone('UTC').strftime('%H:%M %Z %D')
+        verified_multistamp(stamped_template_path, current_time, page_configuration, 12)
       end
     end
 
+    def self.verified_multistamp(stamped_template_path, signature_text, page_configuration)
+      raise StandardError, 'Provided signature was empty' unless signature_text
```

Comment:
It looks like sometimes we pass a `date_title` or `date_text`, not a signature text. Maybe this should be more general language?

Reply:
Good catch, fixed! 👍